src/layouts/ContentWrapper/ContentWrapper.jsx

```diff
@@ -0,0 +1,7 @@
+import MainPage from '../../pages/MainPage/MainPage';
+
+function ContentWrapper() {
+  return <MainPage />;
+}
+
+export default ContentWrapper;
```

Comment:
The same situation like _HeaderWrapper_. I think that it is an extra component(wrapper).